contrib/single-node.sh

```diff
@@ -26,11 +26,12 @@ gaiad genesis collect-gentxs
 
 # Set proper defaults and change ports
 echo "Setting rpc listen address"
-sed -i '' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.gaia/config/config.toml
-echo 2
-sed -i '' 's/timeout_commit = "5s"/timeout_commit = "1s"/g' ~/.gaia/config/config.toml
-sed -i '' 's/timeout_propose = "3s"/timeout_propose = "1s"/g' ~/.gaia/config/config.toml
-sed -i '' 's/index_all_keys = false/index_all_keys = true/g' ~/.gaia/config/config.toml
+# sed -i '' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.gaia/config/config.toml
```

Comment:
These scripts probably need to be redone tbh. Happy to do if the team would like. Maybe another PR

ref what I use: https://github.com/CosmosContracts/juno/blob/main/scripts/test_node.sh

Reply:
We have this in the backlob but we haven't started refactoring yet: https://github.com/cosmos/gaia/issues/2950

We can add refactoring this script to that issue and do them all in a different PR.